_posts/2021-08-31-CZI-EOSS4.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Chan Zuckerberg Initiative EOSS4 Award
+---
+
+MDAnalysis is excited to announce that we have been awarded an [Essential Open Source Software for Science][] (EOSS) grant by the Chan Zuckerberg Initiative for our proposal [“MDAnalysis: Faster, extensible molecular analysis for reproducible science”][].
+
+Over the next two years, this award will fund work on two major aims that will improve MDAnalysis for our users and make it easier for scientists to contribute code, with a view towards increasing code reuse and interoperability and increasing reproducibility:
+
+
+## (1) Improving the performance of essential core components by rewriting Python code in Cython
+
+As MDAnalysis has now become a mature library with a stable API, we want to focus on improving the performance of core library components by rewriting essential code in Cython. This will enable us to handle the increasingly large and complex datasets generated to solve the biomedical challenges of tomorrow. 
+
+
+## (2) Launching the "MDAKits" ecosystem
+
+We will launch the MDAnalysis Toolkits (“MDAKit”) ecosystem. MDAKits are inspired by SciPy’s SciKits. An MDAKit will contain code based on MDAnalysis that solves specific scientific problems. An MDAKit can be written by anyone and hosted anywhere. In order to streamline the process we will provide a cookiecutter-based template repository. MDAKits will have a well-defined API, have documentation, will be registered with MDAnalysis, and will be continuously tested  for compatibility with the current version of MDAnalysis. As part of the registration process, code will be reviewed by MDAnalysis developers and we will encourage the scientific publication of the package, e.g., in the [Journal of Open Source Software][]. We will also refactor a number of analysis modules as separate MDAKits. Overall, the introduction of MDAKits will enable the rapid development and dissemination of new components and tools that can be released independently of the main package. We will soon describe the details of MDAKits in a white paper and seek a discussion with the wider community. Ultimately, MDAKits should help address two long-standing challenges in the biomolecular simulation community: effort duplication and reproducibility.
```

Comment:
```suggestion
We will launch the MDAnalysis Toolkits (“MDAKit”) ecosystem. Inspired by SciPy’s SciKits, an MDAKit will contain code based on MDAnalysis that solves specific scientific problems. An MDAKit can be written by anyone and hosted anywhere. In order to streamline the process we will provide a cookiecutter-based template repository. MDAKits will have a well-defined API, have documentation, will be registered with MDAnalysis, and will be continuously tested for compatibility with the current version of MDAnalysis. As part of the registration process, code will be reviewed by MDAnalysis developers and we will encourage the scientific publication of the package, e.g., in the [Journal of Open Source Software][]. We will also refactor a number of analysis modules as separate MDAKits. Overall, the introduction of MDAKits will enable the rapid development and dissemination of new components and tools that can be released independently of the main package. We will soon describe the details of MDAKits in a white paper and seek a discussion with the wider community. Ultimately, MDAKits should help address two long-standing challenges in the biomolecular simulation community: effort duplication and reproducibility.
```

Reply:
How much of a re-write is this? We already have approved text...

Reply:
Tagging @orbeckst

Reply:
Sorry, it's just grammar.

"MDAKits are inspired by SciPy’s SciKits. An MDAKit will contain code based on MDAnalysis that solves specific scientific problems." -> " Inspired by SciPy’s SciKits, an MDAKit will contain code based on MDAnalysis that solves specific scientific problems."

and I deleted an extra space

Reply:
will merge